spec/signable/query/response_spec.rb

```diff
@@ -1,34 +1,40 @@
-require "spec_helper"
+# frozen_string_literal: true
 
-describe Signable::Query::Response do
+require 'spec_helper'
 
-  describe "#ok?" do
-    let(:response) { Signable::Query::Response.new http_response }
+# rubocop:disable RSpec/VerifiedDoubles
+describe Signable::Query::Response do
+  describe '#ok?' do
     subject { response.ok? }
 
-    context "when code is 200" do
-      let(:http_response) { double 'http_response', code: 200 }
-      it { should be true }
+    let(:response) { described_class.new http_response }
+
+    context 'when code is 200' do
+      let(:http_response) { double code: 200 }
+
+      it { is_expected.to be true }
     end
 
-    context "when code is 202" do
-      let(:http_response) { double 'http_response', code: 202 }
-      it { should be true }
+    context 'when code is 202' do
+      let(:http_response) { double code: 202 }
+
+      it { is_expected.to be true }
     end
 
-    context "when code is !200" do
-      let(:http_response) { double 'http_response', code: 205 }
-      it { should be false }
+    context 'when code is !200' do
```

Comment:
```suggestion
    context 'when code is not 200' do
```
Clearer do you think?